.github/upload-preview.js

```diff
@@ -27,8 +27,8 @@ switch(uploadFolderName) {
     uploadURL += '-a11y.surge.sh';
     break;
   case 'public':
-    if (!prnum && prbranch === 'main') {
-      uploadURL = 'core-staging.patternfly.org';
+    if (!prnum && prbranch === 'v6') {
```

Comment:
Is the name of the new branch going to be `v6-new`? or will you ultimately name it v6?

All the logic in the upload-preview.js, pr-preview.yml, and release.yml are triggered off activiy on the branch named `v6` as you have it here, now.

Reply:
@nicolethoen my thinking is this will just replace the v6 branch, so no release/workflow changes needed. I was thinking to rename the existing branch (v6-old or something), then rename this one to v6. And v6-old is just there in case something happens and there are commits we want to review, so no extra work needed for it.